README.md

```diff
@@ -1,3 +1,48 @@
 # godess-tribes-frontend
 
-This is the repository of Tribes Front-End Project of Team Godess
+# Project Summary
+
+Tribes by GFA-Goddess, this is the front-end part of a strategy game developed by our team as a coursework in Green Fox Academy
+
+This project was generated with Angular CLI version 7.3.9. | Angular 7.2.15. | Node 10.15.3
+
+# Getting Started
+
+$ npm install -g @angular/cli through command line and download VS Code to   your PC from code.visualstudio.com
+
+Install show-hide password module / follow the instructions in the link below:
```

Comment:
I think all the Install happens if you cloned the repo, and type `npm install`

Reply:
Yes please mention `npm install`